Logger/FileLogger.cs

```diff
@@ -0,0 +1,41 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+
+namespace Logger
+{
+    public class FileLogger : BaseLogger
+    {
+        private string fPath;
+        StreamWriter sWriter;
+
+        public FileLogger(string pth)
+        {
+            fPath = pth;
+
+            if(File.Exists(fPath))
+            {
+
+            }
+            else
+            {
+                File.Create(fPath);
+            }
+        }
+
+        public override void Log(LogLevel logLevel, string message)
+        {
+            DateTime dateTime = DateTime.Now;
+            string dTime = dateTime.ToString();
```

Comment:
From the coding guidelines

> DO NOT use abbreviations or contractions within identifier names.

```suggestion
            string time = dateTime.ToString();
```